src/core-sdk-tasks/DownloadFile.cs

```diff
@@ -39,31 +54,57 @@ public override bool Execute()
                 var filePath = Uri.Substring(FileUriProtocol.Length);
                 Log.LogMessage($"Copying '{filePath}' to '{DestinationPath}'");
                 File.Copy(filePath, DestinationPath);
+                return true;
             }
             else
```

Comment:
`else` is unnecessary now, removing helps with the somewhat deep nesting.

Reply:
@dagood - I'm not sure why the else is unecessary. Shouldn't the task still support download from a `file://` location?

Reply:
I mean that since you added the exit with `return true;`, the `else {` and matching `}` can be removed to improve readability. (Keep what's inside, unindent it.) Not sure how that affects `file://` support.